cli/chrome-headless-render-pdf.js

```diff
@@ -31,6 +32,7 @@ if (argv['help'] || !argv['pdf'] || !argv['url']) {
 
 const urls = typeof argv['url'] === 'string' ? [argv['url']] : argv['url'];
 const pdfs = typeof argv['pdf'] === 'string' ? [argv['pdf']] : argv['pdf'];
+const windowSize = typeof argv['window-size'] === 'string' ? argv['window-size'].split(',') : argv['window-size'];
```

Comment:
I thought it might be best to represent the width and height as separate variables in case we needed that later. E.g. suppose we want to output a verbose debugging message that indicates the height and width as "height: 800 , width: 600". It would be better to have split the argument already. Is that reasonable?